lib/CompilerPluginSupport/CMakeLists.txt

```diff
@@ -82,17 +88,17 @@ if(SWIFT_SWIFT_PARSER)
       DESTINATION "bin"
       COMPONENT compiler
     FRAMEWORK
-      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}"
+      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/host"
       COMPONENT compiler
     LIBRARY
-      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}"
+      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/host"
       COMPONENT compiler
     ARCHIVE
-      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}"
+      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/host"
       COMPONENT compiler)
 
```

Comment:
Similarly, we need `RUNTIME DESTINATION` here.

Reply:
Alright, we'll need to do this over in swift-syntax as well. I'd actually rather start there, because `swift_CompilerPluginSupport` might not stick around.